src/EditorFeatures/Core.Wpf/InlineDiagnostics/InlineDiagnosticsAdornmentManager.cs

```diff
@@ -152,6 +152,11 @@ protected override void AddAdornmentsToAdornmentLayer_CallOnlyOnUIThread(Normali
 
                     var viewLine = viewLines.GetTextViewLineContainingBufferPosition(mappedPoint);
 
+                    if (viewLine is null)
```

Comment:
needs docs on when this could happen.

Reply:
could we push this logic into ShouldDrawTag?

Reply:
Is the scenario where this is reachable applicable to this call?

https://github.com/dotnet/roslyn/blob/605765f4c36a24533f2f5340ad569e1d81c92763/src/EditorFeatures/Core.Wpf/StringIndentation/StringIndentationAdornmentManager.VisibleBlock.cs#L55-L56